src/sentry/templates/sentry/emails/verification-code.txt

```diff
@@ -0,0 +1,7 @@
+You've initiated an account merger which requires verification. Please use the code below to confirm:
+
+{{code}}
+
+This code expires in 30 minutes.
```

Comment:
Should `30` be a variable? If you have to change `TOKEN_MINUTES_VALID` your email copy won't be accurate.